CONTRIBUTING.md

```diff
@@ -2,7 +2,7 @@
 
 When contributing to Kubeapps, please first discuss the change you wish to make via an issue with this repository before making a change.
```

Comment:
I'd add here that we have delegated the chart distribution to the official BAC and thereby the PRs and issues should be created in the bitnami/charts repo.